### ISSUE TITLE
AV: investigate moving focus to SMP play button

### ISSUE THREAD

**Issue by lizcameron** [bbc/simorgh#3674]
**Is your feature request related to a problem? Please describe.**
At the moment, Focus doesn't land on the play button on the AV player so users can't use focus to control the player. 

Currently when a user clicks the play button the placeholder, they have to tab once before they have focus on the play button of SMP. There should be no tab required, focus should automatically move to the SMP play/pause button.

**Describe the solution you'd like**
Investigate whether it is possible to move the focus from the placeholder image 'play button' to the SMP play button, when media (audio or video) autoplays. I.e. as soon as the placeholder image has been clicked and the second iframe has loaded with the SMP player.
If we can't focus the play button in the SMP player, then we should focus the SMP iframe or the embed iframe. Once we know what's technically possible, we should check the solution with UX.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
LINK to how AV works doc - AV component using Flash?

**Comment by lizcameron** [comment]
Following discussion with the SMP team: they do not recommend moving the focus automatically/out of the user's control. The issue described above is caused by introducing a play button on the placeholder image, in addition to SMP's play button - it is not thought there is a resolution to this whilst we are still using the placeholder image.

Potential future resolution if placeholder is in use longer term: Once the flash players disappear, then SMP expect to be able to support moving focus to the player fairly easily.

**Comment by lizcameron** [comment]
Closing for now as the AV player is technically accessible. This issue should be discussed with UX when we work on the player again to improve the UX for screenreader users.
